mocks/handlers/councilHandler.js

```diff
@@ -0,0 +1,430 @@
+import {graphql} from 'msw';
+
+export const councilHandler = graphql.query('getCouncil', (_, res, ctx) => {
+  return res(
+    ctx.data({
+      substrateChainCouncil: {
+        __typename: 'SubstrateChainCouncil',
+        members: [
+          {
+            __typename: 'SubstrateChainCouncilMember',
+            account: {
+              __typename: 'SubstrateChainAccount',
+              address: '1hCMdtRsaRA4ZTEKpPKPvEjK9rZpGhyFnRHSDhqFMCEayRL',
+              display: 'RTTI-5220 (POLKADOT)',
+            },
+            backing: '217343856918926861',
+            formattedBacking: '21.7343  MDOT',
+            voters: [
```

Comment:
Let's reduce the items in the voters list to 2 or 3. We mainly care about the shape of the data rather the exact data.